necroHeal.lic

```diff
@@ -0,0 +1,43 @@
+=begin
+
+  Documentation: https://elanthipedia.play.net/Lich_script_development#necroheal
+  ## ** Placement comments here until finalized and added to pedia.
+
+  Necromancer stand-alone healing script
+  
+  Uses all settings defined in HEAL YAML FILE (default)
+  Consume logic via combat trainer
+  
+  TO DO:
+  - See pull request #1462 for harvested materials.
+  - See if there is a background way to use common-healing to determine wound status

```

Comment:
Generally speaking, sub scripts should always function stand alone, so people who just want to necro-heal can, but tying it into safe-room is quite useful since it's basically the necro version of automated healing.